.github/workflows/_cleanup.yml

```diff
@@ -6,7 +6,34 @@ on:
     - cron: "0 0 * * *"
 
 jobs:
-  remove-workload-and-cluster:
+  cleanup_registry:
+    name: Cleanup Container Registry
+    runs-on: ubuntu-latest
+    steps:
+      - name: Log into Azure
+        run: |
+          az login --service-principal \
```

Comment:
Should the `az login` use federated login credentials like [Bryce's PR](https://github.com/microsoft/confidential-container-demos/pull/24/files)?

Reply:
His PR will replace that once it's merged. I believe they're the same lines he's changing.

Reply:
I think they're different. His changes the login in the job right below (remove-workload-and-cluster).